src/sbw.erl

```diff
@@ -118,7 +118,8 @@ set_multipart(PostParams, PostFiles, Wrapper) ->
 cache_headers(Wrapper) ->
     Mod = Wrapper#sbw.mod,
     Req = Wrapper#sbw.req,
-    FormattedHeaders = [normalize_header({K,V}) || {K,V} <- Mod:headers(Req), V=/=undefined],
+    Fun = fun(K,V) when V=/=undefined -> normalize_header({K,V}) end,
```

Comment:
As a little bit of code review, this construct here ended up producing maps that looked like:

`#{K => {K ,V}}`

rather than the desired

`#{K => V}`

As a result, you needed to strip out the K from the `maps:find` result on line 218. (the `_Key` match on line 220).  I've tweaked this accordingly, but kept the conversion to maps, and have to make the necessary changes to Nitrogen to account for this.

Reply:
ok